samples/ProjectGrandmaster.Unity/Assets/Script/Board/BoardInformation.cs

```diff
@@ -0,0 +1,633 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using System;
+using Microsoft.MixedReality.USYD.ChessPiece;
+using UnityEngine.UI;
+using TMPro;
+
+namespace Microsoft.MixedReality.USYD.Board
+{
+    public class BoardInformation : MonoBehaviour
+    {
+        enum Colours { White, Black };
+        Colours turn;
+
+        GameObject[,] board;
```

Comment:
Could you add the default modifier for each one; I think for fields it's private.